File: solutions/1878-check-if-array-is-sorted-and-rotated/solution.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int breakcount = 0;
        if(nums.size()<=1)return true;

        for(int i=1;i<nums.size();i++){
            if(nums[i-1]>nums[i]){
                breakcount++;
            }
        }
        if(nums[0]<nums[nums.size()-1]){
            breakcount++;
        }
        if(breakcount>1){
            return false;
        }
        return true;
    }
};

File: solutions/1878-check-if-array-is-sorted-and-rotated/solution.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        breakcount=0
        if len(nums)<=1:
            return True
        for i in range(1,len(nums)):
            if nums[i-1]>nums[i]:
                breakcount+=1
        if nums[0]<nums[-1]:
            breakcount+=1
        if breakcount>1:
            return False
        return True
        
